LocalSettings.d/production/hooks.php

```
@@ -1,16 +1,21 @@
 <?php
 use MediaWiki\Title\Title; 
 use MediaWiki\MediaWikiServices;
+use MediaWiki\Parser\Parser;
+
+$wgHooks['ParserBeforeInternalParse'][] = static function ( Parser &$parser, &$text ) {
+	if ( $text !== '' ) {
+		return;
+	}
+	$page = $parser->getPage();
+	$title = Title::newFromPageReference( $page );
+	if ( $title->getContentModel() !== CONTENT_MODEL_WIKITEXT || !$title->exists() ) {
```

PHPStan (2.1.46)

[error] Constant CONTENT_MODEL_WIKITEXT not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)